BUILDING.md

```diff
@@ -146,7 +146,11 @@ platforms. This is true regardless of entries in the table below.
   systems. Issues that only reproduce on WSL should be reported in the
   [WSL issue tracker](https://github.com/Microsoft/WSL/issues). Running the
   Windows binary (`node.exe`) in WSL is not recommended. It will not work
-  without workarounds such as stdio redirection.
+  without workarounds such as stdio redirection. 
+  If you running into `No module named 'distutils.spawn'` 
+  Error when executing `./configure`, please try `sudo apt install python3-distutils -y` 
+  or `python3 -m pip install setuptools`.
```

Comment:
Can we please reverse these two?  __pip__ does not require sudo and works on non-Ubuntu platforms.

Reply:
distutils is kinda legacy: https://docs.python.org/3/distutils/apiref.html#distutils.core.setup

Reply:
> Can we please reverse these two? **pip** does not require sudo and works on non-Ubuntu platforms.

Just leave the issue link ?

Reply:
I think the text above is correct but just put the __pip__ solution __first__ and the __apt__ solution __second__.

Reply:
Got it :)